client/app/scripts/superdesk-monitoring/monitoring.js

```diff
@@ -546,7 +546,6 @@
                                 list.style.height = (scope.total * ITEM_HEIGHT) + 'px';
                             }
 
-                            list.style.paddingTop = (from * ITEM_HEIGHT) + 'px';
```

Comment:
@petrjasek Can you please check this, you worked on it. This padding top was problem, but I'm not sure why is this padding for, maybe I broke something somewhere else..